workspace-manager-clienttests/README.md

```diff
@@ -61,6 +72,35 @@ export TEST_RUNNER_SERVER_SPECIFICATION_FILE="workspace-dev.json"
 Each service account JSON files in the resources/serviceaccounts directory of this project specifies a default file
 path for the client secret file. This default path should match where the render-config.sh script puts the secret.
 
+#### Test runner SA keys
+The test runner SA key is necessary for uploading test results to Google Buckets and BigQuery. 
+Each environment has its own test runner SA key with the right permissions for uploading test results.
+The provisioning of the test runner SA keys will be done with `Terraform` in collaboration with `devOps`.
+Additional `k8s` permission will allow test runner to manipulate a cluster for resiliency tests.
+
+Here's the vault paths of various test runner SA keys.
+
+Static environments (alpha, dev, perf, qa, staging)
+One test runner SA key for each static environment.
+`secret/dsde/terra/kernel/<env>/common/testrunner-sa`
+
+`<env> = { alpha, dev, perf, qa, staging }`
+
+Preview environments
+One common test runner SA key for all preview environments.
+`secret/dsde/terra/kernel/integration/preview/testrunner-sa`
+
+#### Google Buckets for uploading test runner results
+Each Terra service should have its own bucket for uploading test runner results.
+
+For example,
+`gs://broad-dsde-dev-workspace-manager-testrunner-results`
+
+`broad-dsde-dev` identifies the project name or target environment
+`workspace-manager` identifies the target service as `terra-workspace-manager`
+
+NB: A Google Bucket name has a limit of up to 63 characters.
```

Comment:
Nit: I think this would be better placed in the TestRunner README. It's not really WSM-specific.

Reply:
I added this comment to the Google [doc](https://docs.google.com/document/d/1gYqlJTpC4C4S1K0pYikeNJx-HWNZAZJYybinmuqcSTI/edit#heading=h.8w6xdoo0wwcv) to track it as future work.